### PR TITLE
chore: update 1.3.0 cn download page

### PR DESCRIPTION
**Pull request by sheli00**
## Purpose of the PR

- update download page

### CODE REVIEW

File: content/cn/docs/download/download.md [hunk 1]
```diff
@@ -1,25 +1,28 @@
...
 linkTitle: "Download"
 weight: 2
 ---
 
-### Latest version
 
-The latest HugeGraph: **1.3.0**, released on _2024-04-01_([how to build from source](/docs/quickstart/hugegraph-server/)).
+> Instructions
+> 
+> - 请从下表中选择HugeGraph版本下载，推荐使用最新版本，运行时选择 Java11。
```
Comment (imbajin) [inline content/cn/docs/download/download.md:10]:
```suggestion
> - 推荐使用最新版本的 HugeGraph 软件包，运行时环境请选择 Java11
```

File: content/cn/docs/download/download.md [hunk 2]
```diff
@@ -1,25 +1,28 @@
...
+> - 验证下载版本，请使用相应的哈希（sha512）、签名和 [项目签名验证 KEYS](https://downloads.apache.org/incubator/hugegraph/KEYS)。
+> - 检查哈希（sha512）、签名的说明显示在 [验证说明](/docs/contribution-guidelines/validate-release/) 页面。（[官方验证](https://www.apache.org/dyn/closer.cgi#verify)）
```
Comment (imbajin) [inline content/cn/docs/download/download.md:12]:
```suggestion
> - 检查哈希（SHA512）、签名的说明在 [验证说明](/docs/contribution-guidelines/validate-release/) 页面, 也可参考（[ASF官方说明](https://www.apache.org/dyn/closer.cgi#verify)）
```

File: content/cn/docs/download/download.md [hunk 3]
```diff
@@ -1,25 +1,28 @@
...
+> 
+> 最新版本: **1.3.0**, 2024-04-01 发布([怎么从源码构建](/docs/quickstart/hugegraph-server/))。[发布记录](/docs/changelog/hugegraph-1.3.0-release-notes/)
```
Comment (imbajin) [inline content/cn/docs/download/download.md:14]:
```suggestion
> 最新版本: **1.3.0**, 2024-04-01 发布 ([从源码构建](/docs/quickstart/hugegraph-server/))。[发布记录](/docs/changelog/hugegraph-1.3.0-release-notes/)
```

然后把它放到下一个章节里, 不放在引用中
image

File: content/cn/docs/download/download.md [hunk 4]
```diff
@@ -29,18 +32,22 @@ The latest HugeGraph: **1.3.0**, released on _2024-04-01_([how to build from sou
...
 
-注: 
-1. 请大家尽早迁移到最新 release 版本上, 社区将不再维护 `1.0.0` 前的旧版本(非 ASF 版本)
-2. `1.3.0` 是最后一个兼容 Java8 的主版本, 请尽早使用/迁移运行时为  `Java11` (低版本 Java 有潜在更多的 SEC 风险和性能影响)
+> 注: 
+> 1. 请大家尽早迁移到最新 release 版本上, 社区将不再维护 `1.0.0` 前的旧版本(非 ASF 版本)
+> 2. `1.3.0` 是最后一个兼容 Java8 的主版本, 请尽早使用/迁移运行时为  `Java11` (低版本 Java 有潜在更多的 SEC 风险和性能影响)
+
 
+### 过时版本
```
Comment (imbajin) [inline content/cn/docs/download/download.md:50]:
```suggestion
---

### 过时版本
```

File: content/cn/docs/download/download.md
Comment (imbajin) [inline content/cn/docs/download/download.md:1]: Please also sync the EN doc, THX

Comment (sheli00) [reply content/cn/docs/download/download.md:1]: sure, finish it later

File: content/cn/docs/download/download.md [hunk 5]
```diff
@@ -29,18 +33,24 @@ The latest HugeGraph: **1.3.0**, released on _2024-04-01_([how to build from sou
...
-1. 请大家尽早迁移到最新 release 版本上, 社区将不再维护 `1.0.0` 前的旧版本(非 ASF 版本)
-2. `1.3.0` 是最后一个兼容 Java8 的主版本, 请尽早使用/迁移运行时为  `Java11` (低版本 Java 有潜在更多的 SEC 风险和性能影响)
+> 注: 
+> 1. 请大家尽早迁移到最新 release 版本上, 社区将不再维护 `1.0.0` 前的旧版本(非 ASF 版本)
+> 2. `1.3.0` 是最后一个兼容 Java8 的主版本, 请尽早使用/迁移运行时为  `Java11` (低版本 Java 有潜在更多的 SEC 风险和性能影响)
+
+
+---
 
+### 过时版本
```
Comment (imbajin) [inline content/cn/docs/download/download.md:53]: image

加个 details 标签把这部分默认折叠一下 (如果可以的话)

File: content/cn/docs/download/download.md [hunk 6]
```diff
@@ -1,25 +1,29 @@
...
 ---
 
-### Latest version
 
-The latest HugeGraph: **1.3.0**, released on _2024-04-01_([how to build from source](/docs/quickstart/hugegraph-server/)).
+> Instructions
+> 
+> - 推荐使用最新版本的 HugeGraph 软件包，运行时环境请选择 Java11
+> - 验证下载版本，请使用相应的哈希（SHA512）、签名和 [项目签名验证 KEYS](https://downloads.apache.org/incubator/hugegraph/KEYS)
+> - 检查哈希（SHA512）、签名的说明在 [验证说明](/docs/contribution-guidelines/validate-release/) 页面, 也可参考（[ASF官方说明](https://www.apache.org/dyn/closer.cgi#verify)）
```
Comment (imbajin) [inline content/cn/docs/download/download.md:12]:
```suggestion
> - 检查哈希（SHA512）、签名的说明在 [版本验证](/docs/contribution-guidelines/validate-release/) 页面, 也可参考（[ASF验证说明](https://www.apache.org/dyn/closer.cgi#verify)）
```

File: content/cn/docs/download/download.md [hunk 7]
```diff
@@ -1,25 +1,29 @@
...
-| ------------------- | ------------------------------------------------ | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| HugeGraph-Server    | HugeGraph 的主程序                               | [1.3.0](https://downloads.apache.org/incubator/hugegraph/1.3.0/apache-hugegraph-incubating-1.3.0.tar.gz)([备用](https://dlcdn.apache.org/incubator/hugegraph/1.3.0/apache-hugegraph-incubating-1.3.0.tar.gz))                     |
-| HugeGraph-Toolchain | 数据导入/导出/备份, Web 可视化图形界面等工具合集 | [1.3.0](https://downloads.apache.org/incubator/hugegraph/1.3.0/apache-hugegraph-toolchain-incubating-1.3.0.tar.gz)([备用](https://dlcdn.apache.org/incubator/hugegraph/1.3.0/apache-hugegraph-toolchain-incubating-1.3.0.tar.gz)) |
+### 最新版本
 
-> 注: HugeGraph 所有组件版本号已保持一致, `client/loader/hubble/common` 等 maven 仓库版本号同理, 依赖引用可参考[maven 示例](https://github.com/apache/incubator-hugegraph-toolchain#maven-dependencies)
+最新版本: **1.3.0**, 2024-04-01 发布 ([从源码构建](/docs/quickstart/hugegraph-server/)) [发布记录](/docs/changelog/hugegraph-1.3.0-release-notes/)
```
Comment (imbajin) [inline content/cn/docs/download/download.md:17]:
```suggestion
版本: **1.3.0**, 2024-04-01 发布 ([从源码构建](/docs/quickstart/hugegraph-server/)) [版本说明](/docs/changelog/hugegraph-1.3.0-release-notes/)
```

**Review by imbajin — APPROVED** [review]

**Pull request merged**